src/pages/Dashboard/sub-pages/Vaults/VaultsTable/index.tsx

```diff
@@ -372,8 +373,12 @@ const VaultsTable = (): JSX.Element => {
       return <PrimaryColorEllipsisLoader />;
     }
 
-    const handleRowClick = (vaultId: string) => () => {
-      history.push(PAGES.VAULTS.replace(`:${URL_PARAMETERS.VAULT.ACCOUNT}`, vaultId));
+    const handleRowClick = (vaultId: string, collateralToken: string) => () => {
+      const vault = PAGES.VAULT.replace(`:${URL_PARAMETERS.VAULT.ACCOUNT}`, vaultId)
```

Comment:
I wanted to follow the same strategy as previously, so I could only think on this chained replaces.